src/main/java/com/google/cloud/tools/app/api/DefaultConfiguration.java

```diff
@@ -14,9 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.app.impl.config;
-
-import com.google.cloud.tools.app.api.Configuration;
+package com.google.cloud.tools.app.api;
```

Comment:
I don't mind renaming app -> appengine, but in order to be consistent, wouldn't we also want to rename the library itself to appengine-tools-lib-for-java, not to mention the plugins?